File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.Chart.MSTest/Param_ProjectName.Core.Tests.MSTest/Tests_postaction.cs
﻿//{[{
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.MSTest
{
    public class Tests
    {
        //^^
        //{[{

        // Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [TestMethod]
        public async Task EnsureSampleDataServiceReturnsChartDataAsync()
        {
            var actual = await SampleDataService.GetChartDataAsync();

            Assert.AreNotEqual(0, actual.Count());
        }
        //}]}
    }
}

File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.Chart.MSTest._VB/Param_ProjectName.Core.Tests.MSTest/Tests_postaction.vb
﻿'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public Class Tests
    '^^
    '{[{

    ' Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <TestMethod>
    Public Async Function EnsureSampleDataServiceReturnsChartDataAsync() As Task
        Dim actual = Await SampleDataService.GetChartDataAsync()

        Assert.AreNotEqual(0, actual.Count)
    End Function
    '}]}
End Class
